python/lsst/ts/ess/csc/rpi_data_client.py

```diff
@@ -267,71 +268,82 @@ async def connect(self) -> None:
         )
         configuration = {"devices": self.config.devices}
         await self.run_command(command="configure", configuration=configuration)
-        await self.run_command(command="start")
 
     async def disconnect(self) -> None:
         """Disconnect from the RPi.
 
         Always safe to call, though it may raise asyncio.CancelledError
         if the writer is currently being closed.
         """
-        try:
-            if self.connected:
-                await self.run_command(command="stop")
-            if self.connected:
-                await self.run_command(command="exit")
-        except ConnectionError:
-            # The connection was lost.
-            # This is not worth getting upset about.
-            self.log.debug("Connection lost in disconnect")
-        finally:
-            if self.connected:
-                await asyncio.wait_for(
-                    tcpip.close_stream_writer(self.writer), timeout=CONNECT_TIMEOUT
-                )
-            if self.mock_server is not None:
-                await self.mock_server.close()
+        if self.connected:
+            await asyncio.wait_for(
+                tcpip.close_stream_writer(self.writer), timeout=CONNECT_TIMEOUT
+            )
+        if self.mock_server is not None:
+            await self.mock_server.close()
+
+    def get_telemetry_schema(self) -> Dict[str, Any]:
```

Comment:
I suggest making this a static method (`@staticmethod` decorator and eliminate the first argument) class method (`@classmethod` decorator and first argument changes to `cls`). That way it can be called on the class itself or an instance of the class.

Reply:
Done